src/modules/navigator/rtl_direct.cpp

```diff
@@ -293,6 +295,14 @@ void RtlDirect::set_rtl_item()
 			if (_param_rtl_land_delay.get() < -FLT_EPSILON) {
 				mavlink_log_info(_navigator->get_mavlink_log_pub(), "RTL: completed, loitering\t");
 				events::send(events::ID("rtl_completed_loiter"), events::Log::Info, "RTL: completed, loitering");
+
+			} else {
+				/* Set the altitude tracking to best effort but not strictly enforce it */
+				altitude_acceptance_radius = FLT_MAX;
+
+				if (_force_heading) {
```

Comment:
that was a separate issue?

Reply:
Which one?

Reply:
forcing the heading - all good, the change makes sense.

Reply:
perfect thanks for the diligence!